core/util.py

```diff
@@ -59,9 +61,9 @@ def group_by_first_element(tuples: list[tuple]) -> dict[str, list[tuple | Any]]:
 
 
 def get_project_number(project_id):
-    """Extracts the project number from a project ID.
+    """Map project ID to it's on page position using the active projects in g
 
     :param project_id: A project ID code
     :return: project number
     """
-    return int(project_id.split("-")[2])
+    return g.active_project_numbers.index(project_id) + 1
```

Comment:
I don't think this should be used like this here. It's making some test fail due to application context

Reply:
have you removed the old functionality of get_project_number? This seems quite different, do you definitely want to switch it out?

Reply:
The new functionality is very similar, there is just a significant refactor here.